### PR TITLE
feat: updates runtimes in examples and defaults to 2023.09

### PR DESCRIPTION
**Pull request by tnolet**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [ ] CLI
* [ ] Create CLI
* [ ] Test
* [ ] Docs
* [x] Examples
* [ ] Other

## Notes for the Reviewer
Updates the examples to the latest runtime.

### CODE REVIEW

**Review by ferrandiaz — DISMISSED** [review]

**Review by ferrandiaz — COMMENTED** [review]
There is a test failing, on the `deploy.spec.ts` I'm unsure how your changes might have affected that particular test

**Comment by tnolet** [comment]
should fix: https://github.com/checkly/checkly-cli/issues/898

File: packages/cli/src/services/__tests__/project-parser.spec.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { parseProject } from '../project-parser'
 
 const runtimes = {
   2023.02: { name: '2023.02', default: false, stage: 'CURRENT', description: 'Main updates are Playwright 1.28.0, Node.js 16.x and Typescript support. We are also dropping support for Puppeteer', dependencies: { '@playwright/test': '1.28.0', '@opentelemetry/api': '1.0.4', '@opentelemetry/sdk-trace-base': '1.0.1', '@faker-js/faker': '5.5.3', aws4: '1.11.0', axios: '0.27.2', btoa: '1.2.1', chai: '4.3.7', 'chai-string': '1.5.0', 'crypto-js': '4.1.1', expect: '29.3.1', 'form-data': '4.0.0', jsonwebtoken: '8.5.1', lodash: '4.17.21', mocha: '10.1.0', moment: '2.29.2', node: '16.x', otpauth: '9.0.2', playwright: '1.28.0', typescript: '4.8.4', uuid: '9.0.0' } },
```
Comment (ferrandiaz) [inline packages/cli/src/services/__tests__/project-parser.spec.ts:8]: in previous runtime we also supported `mocha` do we want to stop supporting it?

**Review by ferrandiaz — APPROVED** [review]
LGTM

**Pull request merged**